src/OpenLoco/src/Objects/ObjectManager.h

```diff
@@ -145,6 +145,12 @@ namespace OpenLoco::ObjectManager
         ObjectHeader problemObject;
     };
 
+    struct DependentObjects
+    {
+        std::vector<ObjectHeader> required;
+        std::vector<ObjectHeader> willLoad;
```

Comment:
What's the difference between these two?

Reply:
The will load is for regions. It will load all the objects in that vector. I'm not sure why they need to be separated. I guess you can select a region but then deselect some parts of it.